apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -891,182 +883,6 @@ export class ApplicationController {
-    })
-
-    return { controlCode, documentToken }
-  }
-
-  @Scopes(ApplicationScope.write)
-  @Put('applications/:id/uploadSignedFile')
-  @ApiParam({
-    name: 'id',
-    type: String,
-    required: true,
-    description: 'The id of the application which the file was created for.',
-    allowEmptyValue: false,
-  })
-  @ApiOkResponse({ type: UploadSignedFileResponseDto })
-  async uploadSignedFile(
-    @Param('id', new ParseUUIDPipe()) id: string,
-    @Body() input: UploadSignedFileDto,
-    @CurrentUser() user: User,
-  ): Promise<UploadSignedFileResponseDto> {
-    const existingApplication =
-      await this.applicationAccessService.findOneByIdAndNationalId(id, user)
-
-    await this.fileService.uploadSignedFile(
-      existingApplication,
-      input.documentToken,
-      input.type,
-    )
-
-    this.auditService.audit({
-      auth: user,
-      action: 'uploadSignedFile',
-      resources: existingApplication.id,
-      meta: { type: input.type },
-    })
-
-    return {
-      documentSigned: true,
-    }
-  }
-
-  @Scopes(ApplicationScope.read)
-  @Get('applications/:id/:pdfType/presignedUrl')
-  @ApiParam({
-    name: 'id',
-    type: String,
-    required: true,
-    description: 'The id of the application which the file was created for.',
-    allowEmptyValue: false,
-  })
-  @ApiOkResponse({ type: PresignedUrlResponseDto })
-  async getPresignedUrl(
-    @Param('id', new ParseUUIDPipe()) id: string,
-    @Param('pdfType') type: PdfTypes,
-    @CurrentUser() user: User,
-  ): Promise<PresignedUrlResponseDto> {
-    const existingApplication =
-      await this.applicationAccessService.findOneByIdAndNationalId(id, user)
-    const url = await this.fileService.getPresignedUrl(
-      existingApplication,
-      type,
-    )
-
-    this.auditService.audit({
-      auth: user,
-      action: 'getPresignedUrl',
-      resources: existingApplication.id,
-      meta: { type },
-    })
-
-    return { url }
-  }
-
-  @Get('applications/:id/attachments/:attachmentKey/presigned-url')
-  @Scopes(ApplicationScope.read)
-  @Documentation({
-    description: 'Gets a presigned url for attachments',
-    response: { status: 200, type: PresignedUrlResponseDto },
-    request: {
-      query: {},
-      params: {
-        id: {
-          type: 'string',
-          description: 'application id',
-          required: true,
-        },
-        attachmentKey: {
-          type: 'string',
-          description: 'key for attachment',
-          required: true,
-        },
-      },
-    },
-  })
-  async getAttachmentPresignedURL(
-    @Param('id', new ParseUUIDPipe()) id: string,
-    @Param('attachmentKey') attachmentKey: string,
-    @CurrentUser() user: User,
-  ): Promise<PresignedUrlResponseDto> {
-    const existingApplication =
-      await this.applicationAccessService.findOneByIdAndNationalId(id, user)
-
-    if (!existingApplication.attachments) {
-      throw new NotFoundException('Attachments not found')
-    }
-
```

Comment:
Are these methods used for normal attachments also?

Reply:
Re-added those